api-spec/protobuf/tdex-daemon/v2/feeder.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+package tdex_daemon.v2;
+
+import "tdex/v1/types.proto";
+import "tdex-daemon/v2/types.proto";
+
+// FeederService is the service to manage price feeds
+service FeederService {
```

Comment:
currently missing `ListPriceFeeds` rpc to let the operator list the price feeds of all markets.

Reply:
Done